features/step_definitions/policy_area_steps.rb

```diff
@@ -20,5 +22,8 @@
 end
 
 Then(/^a policy area called "(.*?)" is published to publishing API$/) do |policy_area_name|
```

Comment:
This statement no longer matches what the code is doing as it's no longer just publishing to publishing API. Maybe the feature would better describe what we expect the code to do if there were two separate steps, e.g. 

```
Scenario: Creating a policy area
  When I create a policy area called "Climate change"
  Then there should be a policy area called "Climate change"
  And a policy area called "Climate change" is published to publishing API
  And a policy area called "Climate change" is indexed for search
```